lib/B/C/OverLoad/B/PMOP.pm

```diff
@@ -122,9 +122,7 @@ sub do_save {
             $initpm->sadd( "PM_SETRE(%s, ReREFCNT_inc(PM_GETRE(%s))); /* %s */", $sym, $pre_saved_sym, $comment );
         }
         else {
-            my ( $cstr, undef, undef ) = savecowpv($re);
-
-            $initpm->sadd( "PM_SETRE(%s, CALLREGCOMP(newSVpvn_flags(%s, %s, SVs_TEMP|%s), 0x%x));", $sym, $cstr, $relen, $utf8 ? 'SVf_UTF8' : '0', $pmflags );
+            $initpm->sadd( "PM_SETRE(%s, CALLREGCOMP(newSVpvn_flags(%s, %s, SVs_TEMP|%s), 0x%x));", $sym, $qre, $relen, $utf8 ? 'SVf_UTF8' : '0', $pmflags );
```

Comment:
we're moving from cstr to qre? Was it a bug?

Reply:
that s the revert of 56cbfc58b4a0b3639ffb622e2781e7e58e1fb065 as mentionned in the message